Sources/MapboxCoreNavigation/PassiveLocationManager.swift

```diff
@@ -190,6 +161,39 @@ open class PassiveLocationManager: NSObject {
         NotificationCenter.default.removeObserver(self)
     }
     
+    // MARK: Accessing Relevant Routing Data
```

Comment:
jazzy uses marks to generate named headings in the published API reference, which are very useful for grouping and linking to related functionality. We don’t have nearly enough marks of this sort in the codebase, but there are examples of it in SpeedLimitView for instance. The Title Case is annoying, but it’s the style Apple uses in their documentation. I’m open to changing it to Sentence case for consistency with internal-only marks.

Reply:
#3180 tracks adding more developer-facing mark comments.